activerecord/lib/active_record/railties/databases.rake

```diff
@@ -158,6 +158,21 @@ db_namespace = namespace :db do
     puts "Current version: #{ActiveRecord::Migrator.current_version}"
   end
 
+  namespace :version do
+    desc 'Prints the current schema version number'
+    task :number => [:environment, :load_config] do
+      puts ActiveRecord::Migrator.current_version
+    end
+
+    desc "Prints the file path of current schema's migration"
+    task :file => [:environment, :load_config] do
+      current_version = ActiveRecord::Migrator.current_version.to_s
+      migration_file = Dir.glob("#{ActiveRecord::Migrator.migrations_path}/*.rb")
+                         .find { |migration| migration.include?(current_version) }
+      puts "Current version: #{migration_file || current_version}."
+    end
```

Comment:
If I understand the usage, it might be more practical to display the filename without a trailing `.`